yt/utilities/amr_kdtree/amr_kdtools.py

```diff
@@ -3,34 +3,39 @@
 from yt.funcs import mylog
 
 
-def receive_and_reduce(comm, incoming_rank, image, add_to_front):
+def receive_and_reduce(comm, incoming_rank, image, add_to_front, use_opacity):
```

Comment:
It's not clear that this function is private, so I'd suggest having a default value here too.
Also, I'd rather we avoid introducing boolean arguments in a positionally-allowed fashion
```suggestion
def receive_and_reduce(comm, incoming_rank, image, add_to_front, *, use_opacity=True):
```